modules/installing-ocp-agent-inputs.adoc

```diff
@@ -36,12 +51,6 @@ $ sudo dnf install /usr/bin/nmstatectl -y
 $ mkdir ~/<directory_name>
 ----
 
-+
-[NOTE]
-====
-This is the preferred method for the Agent-based installation. Using {ztp} manifests is optional.
-====
```

Comment:
I tracked down the creation of this note to [this comment thread here](https://github.com/openshift/openshift-docs/pull/48673#discussion_r997223735). It doesn't make sense to place this note under the step for creating the installation directory, and not specifying what "this" preferred method is seems confusing.

So I moved this note to this section's description. Considering that, since adding this note, I've renamed this procedure to mention "preferred inputs" anyways, I could even delete the note entirely. I'll leave that up to SME and QE to decide.